content/community/news-events.md

```diff
@@ -35,7 +35,6 @@ abstract: "Get the latest news and info about upcoming events"
 | Event | Date | Description |
 | -- | -- | -- |
 | -- | -- | -- |
```

Comment:
Not sure when these showed up in the code, but there should only be one row of these dashes

```
| -- | -- | -- |
```

Same with the table of past events below

Reply:
Just pushed this fix @geealbers. Ok to merge?